spacy/vocab.pyx

```diff
@@ -424,6 +417,14 @@ cdef class Vocab:
             orth = self.strings.add(orth)
         return orth in self.vectors
 
+    def load_lookups(self, lookups):
```

Comment:
I guess technically, this should be called `add_lookups`? Because it doesn't really load anxthing and takes an already loaded `Lookups` object?

Reply:
I guess `set_lookups` then. Or we could make `lookups` a property. The `setter` is still weird because you layer the partial `NORM` calls and you can't really undo it, just add on top.